<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="JEFTestSets" failures="1" tests="2" time="0.004" errors="0">
  <properties/>
  <testcase name="Method1" time="0.002" classname="com.hp.cloudprint.Testcase1"/>
  <testcase name="Method1" time="0.0" classname="com.hp.cloudprint.Testcase2">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;false&amp;gt; but was:&amp;lt;true&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<false> but was:<true>
	at com.hp.cloudprint.Testcase2.Method1(Testcase2.java:13)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 25 stack frames]]>
    </failure>
  </testcase>
</testsuite>
